apps/fabric-website/package.json

```diff
@@ -71,8 +71,8 @@
     "color-functions": "1.1.0",
     "json-loader": "^0.5.4",
     "load-themed-styles": "^1.1.0",
-    "office-ui-fabric-core": ">=7.1.2 <8.0.0",
-    "office-ui-fabric-react": ">=4.35.2 <5.0.0",
+    "office-ui-fabric-core": "^7.2.0",
+    "office-ui-fabric-react": "^4.35.2",
```

Comment:
I think to update this, you need to figure out where the code is that reads the version, and have it read the version from office-ui-fabric-react/package.json.